site/source/docs/api_reference/vr.h.rst

```diff
@@ -0,0 +1,378 @@
+.. _vr-h:
+
+====
+vr.h
+====
+
+The C APIs in `vr.h <https://github.com/kripken/emscripten/blob/master/system/include/emscripten/vr.h>`_ provide basic interfaces for interacting with WebVR from Emscripten.
+
+.. contents:: Table of Contents
+	:local:
+	:depth: 1
+
+.. _test-example-code-vr-api:
+
+Test/Example code
+-----------------
+
+The vr test code demonstrates how to use this API:
+
+	- `test_vr.c <https://github.com/kripken/emscripten/blob/master/tests/test_vr.c>`_
+
+.. _functions-vr-api:
+
+Functions
+---------
+
+Initialization
+==============
+
+.. c:function:: int emscripten_vr_init()
```

Comment:
I'll put in some review comments here, even though I do see this was already merged (which is fine) - I am very late to the review party here.

We would ideally favor returning an `EMSCRIPTEN_RESULT` rather than an `int` in these types of success/fail scenarios.

Reply:
`EMSCRIPTEN_RESULT` is currently part of the html5 API.

While on the one hand, the common use of the vr API will require using the html5 API anyway (for a user-gesture callback to get permission to present), I don't know how much of a good idea is to have vr.h include html5.h.
There are some WebVR specific events like `"onvrdisplayconnect"` or `"onvrdisplayblur"`, which are currently not implemented, but later could or should using the html5 API?

I could also extract the emscripten result definitions into a separate header.
What do you think?

Reply:
@juj I would highly appreciate your input on this to be able to wrap up #5640 
I saw the WebVR API has been released in the meanwhile, which makes me a little uncomfortable with breaking the return values in this way now...